analytic_amount_security/readme/CONFIGURE.md

```diff
@@ -0,0 +1,14 @@
+By itself, the module respects the original permissions. All internal users will be able to access the amounts of the analytic lines to which they have access.
+
+To constrain which groups are allowed to see those costs, you can either:
+
+- Add or remove users:
+
+  1. Go to *Settings > Users & Companies > Groups* and locate the group **Allow to read analytic costs**.
+  2. Modify the users in the *Users* tab.
```

Comment:
suggestion: I think it's simpler to enable developer mode (which you have to do anyway) and find the new group under technical section:

![image](https://github.com/user-attachments/assets/e7be0647-80dd-4a4f-9ec3-980eab25f9ca)